src/api/db/migrate/20190520152721_add_index_to_commit_activities.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToCommitActivities < ActiveRecord::Migration[5.2]
+  def change
+    add_index :commit_activities, [:user_id, :date]
```

Comment:
Probably not a big deal, but I would squash this into the first migration.

Reply:
done - as I had to touch the structure.sql anyway for the utf8 fix